### DISCUSSION TITLE
QC's contamination data

### DISCUSSION THREAD

**Discussion by maotian06** [uclahs-cds/pipeline-call-gSNP#76] [Q&A]
@tyamaguchi-ucla 
I am reading my OSCC QC's contamination data, and here is one of the sample outputs:
sample | contamination  |  error
-----|------|----- 
MTOSIMHO000033-N001-A01-F   |    0.12376796182545992  |   0.0074568334509377415

But my other ATC call-gSNP sample is like:
sample | contamination  |  error
-----|------|----- 
ATCWGS21N   |    8.74087704075081E-5  |   9.059114090615396E-6

Based on [GATK's code](https://github.com/broadinstitute/gatk/blob/f4082704f983969cae1333208f7055c7ea9ea1b7/src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/ContaminationModel.java#L164-L192), I think the contamination is based on the ratio of the contamination:
`contaminationEstimate = contaminationOppositeDepth / totalDepthWeightedByOppositeFrequency`

So 0.1 = 10%.

But for the error bar of the contamination level, the output doesn't have the estimate.high and estimate.low.  Can I directly use the `+error / -error` as the error bar?

**Comment by tnyamaguchi** [comment]
According to https://gatk.broadinstitute.org/hc/en-us/articles/360050814292-CalculateContamination, it "calculates **the fraction of reads** coming from cross-sample contamination, given results from GetPileupSummaries". So, the equation makes sense. It looks like the OSCC sample was highly contaminated. (~12%) Any other highly contaminated samples? How about call-sSNV results? They might have shown hypermutation then.

**Reply by maotian06** [reply]
Yes, that means the OSCC data was highly contaminated.
I check my ATC dataset, most of the samples the contamination rate is below 1%. So I guess it is just the OSCC dataset.
Any possible way to save this data?

**Reply by tnyamaguchi** [reply]
I would drop samples with cross individual contamination > 3~4%. Any normal and tumour pairs with less contamination? All samples just have one pair of FASTQs so partial clean-up is not practically possible.
